docs/mods/light_mods.md

```diff
@@ -1,3 +1,125 @@
+It also includes instructions on **replacing the existing resistor** to increase the LED brightness.
+
+The Centauri Carbon includes a MOSFET circuit for a nozzle LED, but the LED itself is unpopulated.  
+The installed current-limiting resistor (100 Ω) results in a dim output, drawing only about 20 mA. With a few quick soldering steps, you can enable and tune this feature.
+
+If you need any support, feel free to [join the Discord](https://discord.gg/t6Cft3wNJ3)  
+
+
+## Requirements  
+
+### Required  
+- **Soldering iron**  
+  Used to solder the LED and (optionally) replace the resistor.
+- **LED:** [WHITE 5730 LED (Cold White)](https://aliexpress.com/item/1005005760920669.html)   
+  Choose the **WHITE 5730** variant — it should match the cold white tone of the chamber LED.
+
+### Optional  
+- **Hot air gun**  
+  Helpful when soldering the 5730 LED, as it has three pads.  
+  The center pad is used for heat transfer to the PCB, but it’s not required for this mod since the LED runs well below its maximum temperature, even in warm chambers.
+- **Resistor:** [27 Ω SMD resistor (0603 size)](https://aliexpress.com/item/1005005677654015.html)  
+  Replaces the stock **100 Ω** resistor to make the LED brighter, increases the current from 20mA to 75mA. The forward voltage increases from 2.72V to 2.92V.  
+  These values will differ slightly from LED to LED. 
+
+
+## Overview  
+
+- The toolhead PCB includes a **nozzle LED circuit** controlled by **MCU pin PC9**.  
+- The **LED + side** connects to the **+5 V rail** (always powered).  
+- The **LED – side** is switched through a MOSFET, controlled by the firmware.  
+- The **current-limiting resistor (R18)** is already present on the PCB but its value limits brightness heavily.
+
+
+## Installation  
+
+1. Locate the **LED pads** on the front edge of the toolhead PCB. 
+![PCB LED pads](assets/PCB_LED.png)  
+*Credit to _sjoerd on the OpenCentauri Discord.*  
+
+2. **Orientation:**  
+   Align the LED so the wider gaps match the PCB pattern before soldering.   
+   Additionally, the PCB has a **white line** marking on the **negative side** for easy orientation.  
+   
+3. Solder the **white 5730 LED** onto the existing footprint:  
+   ![LED placement](assets/LED.png)  
+   *Credit to _sjoerd on the OpenCentauri Discord.*  
+
+    On the LED, the **negative side** can be identified by the **larger gap** between the center pad and the outer pad.  
+
+    It should look something like this:  
+    ![LED Soldered](assets/LED_SOLDERED.png)  
+    *Credit to _sjoerd on the OpenCentauri Discord.*   
+
+4. (Optional) Replace **resistor R18** with **27 Ω** if higher brightness is desired.  
+    ![Resistor R18](assets/R18.jpg)  
+    *Credit to _sjoerd on the OpenCentauri Discord.*  
+
+5. Reassemble the toolhead.  
+
+!!! warning "Note"   
+    We recommend against going below 27 Ω, as the LED may overheat in high-temperature chamber conditions.
+
+
+## Klipper Configuration  
```

Comment:
I would personally leave this out, as it may confuse users on if they have to do something. I would instead put near the top of the page that this mod is compatible with all firmwares on the centauri carbon